src/main/resources/templates/modal/groupingOwnersModal.html

```diff
@@ -0,0 +1,29 @@
+<html xmlns:th="http://www.thymeleaf.org" lang="en">
+<div class="modal-header">
+    <h5 class="modal-title" id="modal-title">Owners</h5>
+</div>
+
+<div class="container modal-table">
+    <table class="table table-sm table-borderless">
+        <thead>
+        <tr class="text-nowrap">
+            <!-- Display table headings -->
+            <th scope="col">Name</th>
+            <th scope="col">UH Username</th>
+            <th scope="col">UH User ID</th>
```

Comment:
Was the order and name of table headings discussed with Michael? In a multi-add modal, it looks like this: 
![image](https://user-images.githubusercontent.com/42422209/233737898-e8b0a828-9fd4-49a7-a8b7-bb1522250796.png)